omoefe-dukuye/SendIT
# #161865975 Set up database

#### What this PR does
If merged, this pull request will
- Set up postgres
- Upgrade previously created endpoints to use persistence

#### How to test manually
- Clone the repo `git clone https://github.com/un-tethered/SendIT.git`
- Checkout the branch `ch-set-up-database-161865975`
- Install Postgresql
- Create a database named `SendIT`
- Run `npm install` to install application dependencies
- Run `babel-node db.js createTables` to create `parcels` table
- Start server with `npm run dev` 
- Test endpoints with Postman
- Restart server and test again to observe persistence.

#### Pivotal tracker story
[#161865975](https://www.pivotaltracker.com/story/show/161865975)
